database/config/manager/manager.yaml

```
@@ -71,6 +71,11 @@ spec:
           value: "24.04"      
         image: controller:latest
         name: manager
+        env:
```

YAMLlint (1.37.1)

[error] duplication of key "env" in mapping

(key-duplicates)